src/views/Home.jsx

```diff
@@ -1,9 +1,12 @@
 import './Home.css';
-import { SingleList } from '../components';
+import { SingleList, ShareListComponent } from '../components';
 import { createList, useAuth, deleteList } from '../api';
 import { Fragment, useState, useEffect } from 'react';
 import { useNavigate } from 'react-router-dom';
 import { useVoiceToText } from '../utils';
+import DeleteIcon from '@mui/icons-material/Delete';
+import KeyboardVoiceIcon from '@mui/icons-material/KeyboardVoice';
+import ShareIcon from '@mui/icons-material/Share';
```

Comment:
<!-- Nitpicks are trivial preference-based requests.  -->
**nitpick ( non-blocking):** <comment> little clean up can get rid of the `ShareIcon` import